app/test_main.py

```diff
@@ -1,3 +1,4 @@
 from app.main import get_human_age
 
-# write your code here
+def test_get_human_age_should_return_animal_age_in_human_years() -> None:
```

Comment:
Consider adding more test cases to cover different scenarios, including edge cases as described in the task description. You can use `@pytest.mark.parametrize` to efficiently test multiple input combinations.